README.md

```diff
@@ -171,6 +171,8 @@ You can deploy this project using:
 
 GitHub Pages
 
+link - https://ricardoegas.github.io/
+
```

Comment:
- [OPTIONAL] You can put the link to the demo in the "Live Demo" section of the readme instead which is the main section for the live demo.
https://github.com/RicardoEgas/MyPortfolio/blob/001ecefb7ff1f8117ec31f9942d767e254400c79/README.md?plain=1#L110-L115